src/app/components/ATIAnalytics/params/index.test.ts

```diff
@@ -271,9 +271,28 @@ describe('ATIAnalytics params', () => {
         undefined,
         homePageAnalyticsData,
       );
-      expect(url).toMatchInlineSnapshot(
-        `"s=598285&s2=atiAnalyticsProducerId&p=kyrgyz.page&r=0x0x24x24&re=1024x768&hl=00-00-00&lng=en-US&x1=[urn%3Abbc%3Atipo%3Atopic%3Acm7682qz7v1t]&x2=[responsive]&x3=[atiAnalyticsAppName]&x4=[pcm]&x5=[http%253A%252F%252Flocalhost%252F]&x7=[index-home]&x8=[simorgh]&x9=[pageTitle]"`,
-      );
+
+      const parsedATIURLParams = new URLSearchParams(url as string);
+
+      const expectedATIURLParams = [
+        ['s', '598285'],
+        ['s2', 'atiAnalyticsProducerId'],
+        ['p', 'kyrgyz.page'],
+        ['r', '0x0x24x24'],
+        ['re', '1024x768'],
+        ['hl', '00-00-00'],
+        ['lng', 'en-US'],
+        ['x1', '[urn:bbc:tipo:topic:cm7682qz7v1t]'],
+        ['x2', '[responsive]'],
+        ['x3', '[atiAnalyticsAppName]'],
+        ['x4', '[pcm]'],
+        ['x5', '[http%3A%2F%2Flocalhost%2F]'],
+        ['x7', '[index-home]'],
+        ['x8', '[simorgh]'],
+        ['x9', '[pageTitle]'],
+      ];
+
+      expect([...parsedATIURLParams]).toEqual(expectedATIURLParams);
```

Comment:
Can we do this instead? 

```suggestion
      expect(parsedATIURLParams).toStrictEqual(expectedATIURLParams);
```

or maybe even

```suggestion
      expect(parsedATIURLParams).toStrictEqual([
        ['s', '598285'],
        ['s2', 'atiAnalyticsProducerId'],
        ['p', 'kyrgyz.page'],
        ['r', '0x0x24x24'],
        ['re', '1024x768'],
        ['hl', '00-00-00'],
        ['lng', 'en-US'],
        ['x1', '[urn:bbc:tipo:topic:cm7682qz7v1t]'],
        ['x2', '[responsive]'],
        ['x3', '[atiAnalyticsAppName]'],
        ['x4', '[pcm]'],
        ['x5', '[http%3A%2F%2Flocalhost%2F]'],
        ['x7', '[index-home]'],
        ['x8', '[simorgh]'],
        ['x9', '[pageTitle]'],
      ]);
```

Reply:
Good point. However, in both suggested scenarios what is received results in an empty object and the test failing.

We're not currently using that method in the other tests so i'm wondering if it's necessary to do so? The `url` that is parsed is serialized to the same string that is passed in, I think. The strict equal does have the added advantage of handling `undefined` scenarios though. 

<img width="1164" alt="image" src="https://github.com/bbc/simorgh/assets/72317734/72a5fa0a-cc73-4b35-b1b4-2be53f011ad3">

Reply:
I don't think we need to be prescriptive about which type of expectation to use in tests - it's really about whatever works in each scenario. If there is a way of getting it to work, such that we're not having to spread the parsedURLParams, that would be good.

Reply:
We've pushed an update which uses object notation instead so it looks cleaner!